yarn.lock

```diff
@@ -1516,10 +1516,10 @@ concat-stream@^1.5.0:
     readable-stream "^2.2.2"
     typedarray "^0.0.6"
 
-conjure-api@^4.3.0:
-  version "4.3.0"
-  resolved "https://registry.yarnpkg.com/conjure-api/-/conjure-api-4.3.0.tgz#c778736f369e7c749a86118e14dab83deb698ace"
-  integrity sha512-AnZe6qkiCRSsfuQMgA556dFs+/DbXpCK2uuhq9bHn36dSbKi+6HUDntpc9jkVPzWOecLXGqy+JAD6IKV8c9Sqw==
+conjure-api@^4.13.0:
```

Comment:
why did we need to bump the conjure API?

Reply:
The old one did not contain the "deprecated" key